desktop/npm-helper.js

```diff
@@ -364,18 +481,20 @@ function updatedFonts () {
       }
     })
 
-  icons['iconfont-proof-new'] = {isFont: true, gridSize: icons['iconfont-proof-good'].gridSize}
-  icons['iconfont-proof-followed'] = {isFont: true, gridSize: icons['iconfont-proof-good'].gridSize}
+  const grids = svgToGridMap()
+  let charCode = 0xe900
```

Comment:
We derive this ourselves since this is based on the order of the project we generate ourselves